float32 yaw

float32 goalX
float32 goalY

float32 errorX
float32 errorY

float32 PerrorX
float32 PerrorY

float32 PvelX
float32 PvelY

float32 VerrX
float32 VerrY
float32 VerrA

float32 CTgainP
float32 CTgainD

float32 ATgainP
float32 ATgainI

float32 ANGgainP
float32 ANGgainI